pyvista/plotting/picking.py

```diff
@@ -120,7 +120,7 @@ def end_pick_helper(picker, event_id):
                     renderer = self.renderers[index]
                     for actor in renderer._actors.values():
                         mapper = actor.GetMapper()
-                        if isinstance(mapper, vtk.vtkDataSetMapper) and mapper.GetInput() == mesh:
```

Comment:
I presume this was also not covered in tests. Otherwise this should've raised, right?

Reply:
Well no idea why the CIs didn't report this but I ran into it locally

Reply:
It's because we're not testing all the picking.  This is hard to test on CI and would be great if we could add this one day.